browser_patches/firefox/patches/bootstrap.diff

```diff
@@ -2471,6 +2483,25 @@ index 0000000000000000000000000000000000000000..13c3cd817b369ed012326b97d03ba6e1
 +    return this._scriptsToEvaluateOnNewDocument;
 +  }
 +
++  addBinding(name, script) {
++    this._bindings.set(name, script);
++    for (const frame of this.frames())
++      this._addBindingToFrame(frame, name, script);
++  }
++
++  _addBindingToFrame(frame, name, script) {
++    Cu.exportFunction((...args) => {
++      this.emit(FrameTree.Events.BindingCalled, {
++        frame,
++        name,
++        payload: args[0]
++      });
++    }, frame.domWindow(), {
++      defineAs: name,
++    });
++    frame.domWindow().eval(script);
```

Comment:
Should we use mainContext.evaluate instead to circumvent CSP? Similar to scripts to evaluate on load.

Reply:
I can't get ahold of it from there!